src/main/java/pl/javastart/task/TaxRates.java

```diff
@@ -0,0 +1,19 @@
+package pl.javastart.task;
+
+public class TaxRates {
+    private double taxRate1 = 0.08;
+    private double taxRate2 = 0.16;
+    private double taxRate3 = 0.23;
```

Comment:
To wygląda mi na stałe. Stałe deklarujemy dodając do nich `static` i `final` i zapisujemy DUZYMI_LITERAMI_Z_PODKRESLNIKAMI